{
  try {
    exec("return (-2147483648) / -1;");
    fail("should have hit exception");
  }
 catch (  ArithmeticException expected) {
  }
  try {
    exec("return (-9223372036854775808L) / -1L;");
    fail("should have hit exception");
  }
 catch (  ArithmeticException expected) {
  }
}
